File: 03.API/src/models/Feedback.js
const { DataTypes } = require('sequelize');
const sequelize = require('../config/database');

const Feedback = sequelize.define('Feedback', {
  id_feedback: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
  id_product: { type: DataTypes.INTEGER, allowNull: false },
  id_login: { type: DataTypes.INTEGER, allowNull: false },
  id_order: { type: DataTypes.INTEGER, allowNull: false }, // ✅ THÊM
  rating: { type: DataTypes.INTEGER, defaultValue: 0 },
  comment: { type: DataTypes.STRING(1000) },
  created_at: { type: DataTypes.DATE, defaultValue: DataTypes.NOW },
  admin_reply: { type: DataTypes.STRING(1000) },
  reply_at: { type: DataTypes.DATE }
}, {
  tableName: 'feedbacks',
  timestamps: false
});

module.exports = Feedback;

File: 01.App/app/src/main/java/com/example/dacnapp/data/model/feedback/Feedback.java
package com.example.dacnapp.data.model.feedback;
import com.example.dacnapp.data.model.product.Product;
public class Feedback {
    public int id_feedback;
    public int id_product;
    public int id_login;
    public int id_order;
    public int rating;
    public String comment;
    public String created_at;
    public String admin_reply;
    public String reply_at;
    public Product Product;
}
